src/map/view.js

```diff
@@ -217,9 +219,56 @@ function getNearestZoomLevelBoxRange(width, height, boxRange) {
 	return newBoxRange;
 }
 
+/**
+ * Calculate bounding box from view for map in EPSG 3857 (Web Mercator) projection
+ * @param center {{lat: Number, lon: Number}} Panther view center
+ * @param boxRange {Number} Panther view box range
+ * @param viewportRatio {Number} Width/height ratio of map window
+ * @param optLat {Number} The latitude used to minimise the distortion between 2D and 3D projection
+ * @return {{minLon: number, maxLat: number, minLat: number, maxLon: number}} Bounding box
+ */
+function getBoundingBoxFromViewForEpsg3857(
+	center,
+	boxRange,
+	viewportRatio,
+	optLat
+) {
+	// convert back the boxRange which was adjusted in map in EPSG 3857 projection
+	// if optLat exists && cos(optLat) !== 0
+	if (optLat && optLat % 90 !== 0) {
+		boxRange /= Math.cos(math.degToRadians(optLat));
```

Comment:
You are modifying input parameter value "boxRange" , so you are not able to get original value passer into getBoundingBoxFromViewForEpsg3857 after function execution. 

It is just small thing, but it could prevent some What and Why.

Reply:
I don't get it. Where I won't be able to get the original value?

Reply:
```
var center = [15,50];
var boxRange = 15000;
var viewportRatio = 0.3;
var bbox = getBoundingBoxFromViewForEpsg3857(center,boxRange,viewportRatio);

console.log(boxRange === 15000)

if(boxRange > 15000) {
//do something
}

```

Reply:
I don't think, that the original value will be changed... https://jsfiddle.net/xwtp0zdc/

Reply:
Your code is more like this: https://jsfiddle.net/wx68r1tb/ , but it works. What? Whay? 

OK, forget it.